example/src/pages/index.tsx

```diff
@@ -108,9 +111,7 @@ const IndexPage = () => {
    */
   const onSwap = useCallback(() => {
     const quoteIdParam = searchParams.get(QueryParams.QuoteId);
-    const quoteId = quoteIdParam
-      ? decodeURIComponent(quoteIdParam)
-      : "84F84F76-FD3A-461A-AF6B-D03F78F7123B";
+    const quoteId = quoteIdParam ? decodeURIComponent(quoteIdParam) : undefined;
```

Comment:
You're right to get rid of this hardcoded value 🙏 
But beware as I think if we don't provide a quoteId to the backend, the call to get the payload will fail. Something to check with backend team.

Reply:
Tested, it works. 
We were not used quoteId in our last versions.